salsah/src/public/js/jquery.searchbox.js

```diff
@@ -206,7 +208,12 @@
 							left: pos.left,
 							top: pos.top
 						}).show();
-						$this.val('');
+						// if the input already has a value and it is valid, back it up
```

Comment:
If I understand correctly, `$this.val()` is the text displayed in the input box.

However, the main thing we are interested in is the Iri of the resource referred to: `$this.data('value', $(this).data('id'));`